set.mm

```diff
@@ -60519,6 +60578,17 @@ in the range of the function (the implication "to the right" is always
   fex $p |- ( ( F : A --> B /\ A e. C ) -> F e. _V ) $=
     ( wf wfn wcel cvv ffn fnex sylan ) ABDEDAFACGDHGABDIACDJK $.
 
+  ${
+    $d A y $.  $d V x y $.  $d W y $.  $d Y y $.
+    mptfvmpt.y $e |- ( y = Y -> M = ( x e. V |-> A ) ) $.
+    mptfvmpt.g $e |- G = ( y e. W |-> M ) $.
+    mptfvmpt.v $e |- V = ( F ` X ) $.
+    $( A mapping as a the function value of a mapping.  (Contributed by AV,
```

Comment:
Maybe: "Value of a function-valued function" ?
(in any case: you have written "a the")

Reply:
Comment corrected/modified.